### PR TITLE
Add and link some icons

### PR DESCRIPTION
**Pull request by AngelaCooljx**
## Description

Add and link some icons, and fix several duplications and restore missing line in previous commits

## Type of change

:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)

### Icons added
* 5G Tile (`moe.xzr.fivegtile`) 
* Aijiasu (`com.fvcorp.android.aijiasuclient`)
* Amap (`com.autonavi.minimap`)
* AliyunDrive (`com.alicloud.databox`)
* AndFTP (`lysesoft.andftp`)
* AnHui 10000 (`com.wondertek.ahPalm10000`)
* App Market (`com.heytap.market`)
* Baidu IME (`com.baidu.input`)
* Baidu NetDisk (`com.baidu.netdisk`)
* bilibili (`tv.danmaku.bili`)
* bilibili (`com.bilibili.app.in`)
* Breeno (`com.heytap.speechassist`)
* Chaoxing (`com.chaoxing.mobile`)
* Clash (`com.github.kr328.clash`)
* Clone Phone (`com.coloros.backuprestore`)
* HeyTap Cloud (`com.heytap.cloud`)
* HeyTap Music (`com.heytap.music`)
* My OnePlus (`com.heytap.vip`)
* OPPO Store (`com.oppo.store`)
* Phone Manager (`com.coloros.phonemanager`)
* Taobao (`com.taobao.taobao`)
* Videos (`com.heytap.yoli`)
* Zhihu (`com.zhihu.android`)
* Bank of China (`com.chinamworld.bocmbci`)
* DSU Sideloader (`vegabobo.dsusideloader`)
* ele.me (`me.ele`)
* ICBC (`com.icbc`)
* JD (`com.jingdong.app.mall`)
* Kernel Flasher (`com.github.capntrips.kernelflasher`)
* Meituan Waimai (`com.sankuai.meituan.takeoutnew`)
* Remote Desktop Manager (`com.devolutions.remotedesktopmanager`)
* Share (`com.hengye.share`)
* Tailscale (`com.tailscale.ipn`)
* WakeUp Schedule (`com.suda.yzune.wakeupschedule`)
* WiFi Moho (`com.ruijie.wifim`)
* WiFi Moho (`cn.com.ruijie.magicbox`)
* Cainiao (`com.cainiao.wireless`)
* DataBackup (`com.xayah.databackup`)
* Devcheck (`flar2.devcheck`)
* DNA-Android (`com.dna.tools`)
* e-CNY (`cn.gov.pbc.dcep`)
* Fake Location (`com.lerist.fakelocation`)
* Fenbi (`com.fenbi.android.servant`)
* GIF Helper (`com.didikee.gifparser`)
* Gongkao Leida (`com.maidu.gkld`)
* Guotai (`com.guotai.dazhihui`)
* Huawei Health (`com.huawei.health`)
* Shortcut Creator (`com.x7890.shortcutcreator`)
* Umetrip (`com.umetrip.android.msky.app`)
* LibChecker (`com.absinthe.libchecker`)
* Litiaotiao (`hello.litiaotiao.app`)
* Magic iPerf (`com.nextdoordeveloper.miperf.miperf`)
* MemoryDetector (`io.github.huskydg.memorydetector`)
* Mirai Plugin for Parabox (`com.ojhdtapp.miraipluginforparabox`)
* MyAndroidTools (`cn.wq.myandroidtools`)
* NetEase Music (`com.netease.cloudmusic`)
* NP Manager (`com.wn.app.np`)
* Parabox (`com.ojhdtapp.parabox`)
* Quark (`com.quark.browser`)

### Icons linked
* Browser (linked `com.heytap.browser` to `@drawable/browser`)
* Calculator (linked `com.coloros.calculator` to `@drawable/calculator`)
* Calendar (linked `com.coloros.calendar` to `@drawable/calendar`)
* Clock (linked `com.coloros.alarmclock` to `@drawable/clock`)
* Compass (linked `com.coloros.compass2` to `@drawable/compass`)
* Files (linked `com.coloros.filemanager` to `@drawable/files`)
* Notes (linked `com.coloros.note` to `@drawable/notes`)
* Recorder (linked `com.coloros.soundrecorder` to `@drawable/recorder`)
* Theme Store (linked `com.heytap.themestore` to `@drawable/themes`)
* Wallet(linked `com.finshell.wallet` to `@drawable/wallet`)

### CODE REVIEW

**Comment by AngelaCooljx** [comment]
This pull request has the same content as #1170, but based on the latest commit to reduce conflicts, and adds ten new icons

File: svgs/np_manager.svg
Comment (Chefski) [inline svgs/np_manager.svg:1]: Not centered. Is it supposed to be off centre a bit?

**Review by Chefski — APPROVED** [review]
I fixed it. Thanks for all the icons :)

**Comment by x9136** [comment]
@AngelaCooljx Hey! Could you tell me where I can download *.apk for DNA-Android (com.dna.tools)?

**Pull request merged**